.env

```diff
@@ -1 +1,7 @@
+# .env for local
+ENVIRONMENT=local
 API_TARGET=http://backend:8000/
+DATABASE_USER=gwells
+DATABASE_PASSWORD=test1
+GWELLS_SERVICE_HOST=db
+GWELLS_SERVICE_PORT=5432
```

Comment:
We should remove the env from git entirely. We can update the `vue.config.js` file to reflect this. 
To keep an easy template I've historically just saved these as `.env.template` instead so the user can copy paste them and add the correct values without fear of being committed